### PR TITLE
Android policy integration

### PR DESCRIPTION
**Pull request by milanperera**

### CODE REVIEW

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+public class PolicyOperationsMapper {
+
+	private static final String TAG = PolicyOperationsMapper.class.getSimpleName();
+	// invalid flag is used to denote operations that are built within agent
+	// thus, it does not have to send to server
+	private static final int INVALID_FLAG = -1;
+
+	public Operation getOperation(org.wso2.mdm.agent.beans.Operation operation) throws AndroidAgentException {
+		switch (operation.getCode()) {
+
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java]: unnecessary empty line

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	private Operation buildPasswordPolicyOperation(Operation operation) {
+		operation.setId(INVALID_FLAG);
+		return operation;
+	}
+
+	private Operation buildWifiOperation(Operation operation) {
+		operation.setId(INVALID_FLAG);
+		return operation;
+	}
+
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java]: Unnecessary empty line

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java (hunk 1 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java]: Removed in PR [#70].

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java (hunk 2 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/PolicyOperationsMapper.java]: Removed in PR [#70].

**Pull request merged**